README.md

```diff
@@ -72,16 +82,31 @@ launch the following actions:
 
 ![vio-architecture-stack](docs/images/edge_orchestrator-actions.png)
 
-## Releases
+## 🛰️ Technology features
+- 🏠 Hosting possibilities :
+  - ☁️ Cloud possibilities with [Azure](https://portal.azure.com/#home) and [GCP](https://cloud.google.com/)
+  - 👮 Float management with [Ansible](https://docs.ansible.com/ansible/latest/index.html)
```

Comment:
"Fleet management" ?

Reply:
**Hosting:**
- on the cloud
- on the edge:
> 1. raspberry
> 2. locally
>         - with anaconda env
>         - with docker

Reply:
C'est même un peu plus complexe que ca:

**Deployment:**

Hub = là où tout remonte > le point de centralisation de la flotte
Host = là où se passe les choses > les éléments de la flotte
> - edge 
> - cloud